[16.0][ADD] shopfloor_reception_product_dimension/mobile

rousseldenis opened the pull request:
Depends on:

- #1004

rousseldenis committed:
[16.0][ADD] shopfloor_reception_product_dimension

rousseldenis committed:
[16.0][ADD] shopfloor_reception_product_dimension_mobile

rousseldenis committed:
[DONT MERGE] test-requirements.txt

lmignon reviewed (changes requested):
IMO, the set dimension step should only be visible if no dimension is set on product. Otherwise, you'll require an additional click on each product you process

rousseldenis committed:
[IMP] shopfloor_reception_product_dimension_mobile: Add weight uom

As operator should know in which unit of measure the weight should be filled in,
add the weight uom as suffix

nicolas-delbovier-acsone reviewed (commented):
LGTM, can we re-open this?
